web/themes/custom/bevan/templates/layout/page.html.twig

```diff
@@ -24,15 +29,16 @@
 
       {% if page.sidebar_first %}
         <div class="govuk-grid-column-one-third">
-          <aside role="complementary">
+          <section role="search" id="search-region">
```

Comment:
So this feels like a fairly big change here, in the main `page.html.twig` template — changing the `page.sidebar_first` to permanently be the `search-region`. This is true on the directory page, but if `page.sidebar_first` ever gets used anywhere else in Connect (I know it's not at the moment, but it may well do in future features) then it may be problematic markup.

Can a different way of doing this be found, some way that is less focussed on the immediate needs of the directory page and has a slightly wider perspective on the use of the `page.html.twig` template in the system?

Reply:
This could be solved by making this change in a `page--directory.html.twig` file so it applies only to the directory page.

Reply:
Yes of course! 🤦‍♂️ Thank you